docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -67,7 +67,7 @@ which cluster the current tab is bound to, and the **Share Feedback** button in
 ## Connecting to an SSH server
 
 1. Open a tab with cluster resources by clicking on the plus symbol at the right end of the tab bar.
-   You can also press <span style="white-space: nowrap;">`Ctrl/Cmd + T`</span> to achieve the same result.
+   You can also press <span style={{ whiteSpace: "nowrap" }}>`Ctrl/Cmd + T`</span> to achieve the same result.
```

Comment:
It looks like it doesn't work with the current version of the docs. If there's no syntax that works both with the current and the previous system than I guess it's fine to leave it broken until the migration happens.

`white-space: nowrap` was added so that individual parts of the shortcut are always on the same line.

<img width="903" alt="shortcut-multiple-lines" src="https://github.com/gravitational/teleport/assets/27113/6fc9a8cf-9f67-46a4-8be1-45fa19a389d4">

https://docs-bcmgu3ox5-goteleport.vercel.app/docs/connect-your-client/teleport-connect/#connecting-to-an-ssh-server

Reply:
@iAdramelk Do you know why inline styles defined as React props aren't rendering? Thanks!